zebra/zebra_rib.c

```diff
@@ -406,7 +406,7 @@ static void nexthop_set_resolved(afi_t afi, const struct nexthop *newhop,
 /* If force flag is not set, do not modify falgs at all for uninstall
    the route from FIB. */
```

Comment:
The comment here is stale now, with `set` removed?

Reply:
fixed.